reference/functions/pgroonga-highlight-html.md

```diff
@@ -96,6 +96,56 @@ SELECT pgroonga_highlight_html('one two three four five',
 
 The keywords are surrounded with `<span class="keyword">` and `</span>`. `<`, `>`, `&` and `"` in `target` is HTML escaped.
 
+**Special Case: Using with `NormalizerTable`**
```

Comment:
Could you create a new subsection like existing `## Practical example: ...` instead of using `**...**` markup?
And could you move this to after existing  `## Practical example: ...` instead of adding to `## Syntax`?

We want to focus on describing syntaxes here.

Reply:
Moved to bottom after `## Practical example: ...`